boot.py

```diff
@@ -10,20 +10,12 @@
 import network
 import time
 
-# custom packages
```

Comment:
same here, this file got updated in #34, I've also removed the import of `esp` to be usable with the Pico